apiproxy/resources/jsc/generate-jwk.js

```diff
@@ -43,10 +43,9 @@
         jwk2.kid = public_key2_kid;
         jwk2.alg = alg;
         jwk2.use = use;
+        certificatelist.keys.push(jwk2);
```

Comment:
This fix is insufficient to fix the issue. As per jwk spec, a key id could be null in a jwk. This fix compares an incorrect condition.